src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml

```diff
@@ -1,9 +1,19 @@
+﻿@inject OrchardCore.Settings.ISiteService siteService
+@using OrchardCore.Entities
+@using OrchardCore.ContentManagement
+@{ 
+    var siteSettings = await siteService.GetSiteSettingsAsync();
+        var themeSettings = siteSettings.As<ContentItem>("ThemeSettings");
```

Comment:
I think `SiteSettings` should be available automatically through `@Site` (comes from the base razor page)